lib/broadway_dashboard.ex

```diff
@@ -146,7 +146,8 @@ defmodule BroadwayDashboard do
           name = inspect(name)
 
           {name,
-           name: name, render: fn -> render_pipeline_or_error(assigns) end, method: :redirect}
+           "navigate",
+           name: name, render: fn -> render_pipeline_or_error(assigns) end}
```

Comment:
To fix formatting issues, you will need to join these two lines :)
Please run `mix format` in your branch.